cfpb/consumerfinance.gov
# Removes unneeded external site gulp task

Also, renames & moves the external-site js file to clarify that it isn't auto-included at a given route by base.html

Testing:
 - Pull and build
 - Navigate to any blog page and click on one of the social share icons
 - The external site page should function as normal
